hardhat.config.ts

```diff
@@ -95,12 +95,32 @@ task("test", "Update fork config")
     await runSuper(taskArguments);
   });
 
+// Pretend that Cancun hardfork was activated at block 0
+const assumeCancun = {
+  hardforkHistory: {
+    cancun: 0,
+  },
+};
+
 const config: HardhatUserConfig = {
   defaultNetwork: "hardhat",
   networks: {
     hardhat: {
       allowUnlimitedContractSize: true,
       loggingEnabled: false,
+      // Forking networks with unknown hardfork activation history causes errors in
+      // new versions of Hardhat. Following https://github.com/NomicFoundation/hardhat/pull/5394,
+      // we assume Cancun hardfork was active from the beginning for all unknown chains
+      chains: {
```

Comment:
What if we move this after the `config` definition, so we can reuse the chain ids, with something like this:

```
// Forking networks with unknown hardfork activation history causes errors in
// new versions of Hardhat. Following https://github.com/NomicFoundation/hardhat/pull/5394,
// we assume Cancun hardfork was active from the beginning for all unknown chains
config.networks.hardhat.chains = {
  ...config.networks.hardhat.chains,
  [config.networks.bscmainnet.chainId]: assumeCancun,
  [config.networks.bsctestnet.chainId]: assumeCancun,
  [config.networks.opbnbtestnet.chainId]: assumeCancun,
  [config.networks.opbnbmainnet.chainId]: assumeCancun,
  [config.networks.arbitrumsepolia.chainId]: assumeCancun,
  [config.networks.arbitrumone.chainId]: assumeCancun,
  [config.networks.opsepolia.chainId]: assumeCancun,
  [config.networks.opmainnet.chainId]: assumeCancun,
}
```

Reply:
Typescript complains that the inner fields are potentially undefined, but I made it more readable here: [280b164](https://github.com/VenusProtocol/vips/pull/401/commits/280b1643a1a770321f516240b1a0a06f71a0e6b3), lmk if it works